wasp-ai/src/server/operations.ts

```diff
@@ -237,3 +237,15 @@ export const getProjectsByUser: GetProjectsByUser<void, Project[]> = async (_arg
     },
   });
 };
+
+export const deleteUser: DeleteUser<{ userId: number }, User> = async (args, context) => {
+  if (!context.user || context.user.id !== args.userId) {
+    throw new HttpError(401, "Not authorized");
+  }
+
+  return await context.entities.User.delete({
+    where: {
+      id: args.userId,
+    },
+  });
+};
```

Comment:
I think you can simplify this in following way:
1. Rename to `DeleteMyself`.
2. Don't accept `userId` as an argument.
3. Use only `context.user.id` to know whom to delete.

Effectively it will work the same, but I think it reduces the space for us to make a mistake here in the code, be it now or in the future when changing something.